src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partner = 'wife';
+
+  if (person.sex === 'f') {
+    partner = 'husband';
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+
+      {person.age ? (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      ) : ''}
```

Comment:
```suggestion
      {person.age && (
        <p className="Person__age">
          {`I am ${person.age}`}
        </p>
      )}
```

Reply:
so did this command, but at first it did not deploy due to "select end of line sequence".
and threw an error.